packaged_releases/HISTORY.md

```diff
@@ -1,6 +1,44 @@
 Release History
 ===============
 
+0.1.6 (2017-01-17)
+------------------
+
+* See the `HISTORY.rst` files in each module for a changelog.  
+
+Source download:  
+https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_0.1.6.tar.gz  
+sha256 bb4bd4694e044bf416941e3afdc505bde7d049bac2e66c8b3fe44c9cb3b57ff8
+
+
+---
+
+<details>
+  <summary>List of component versions in this release (click to expand)</summary><p>
```

Comment:
This is browser dependent. On browsers that don't support it, the full list is shown.